Sources/Types.swift

```diff
@@ -11,8 +11,8 @@ public struct DictionaryType: Type_ {
     public var valueType: Type_
 }
 
-public struct FunctionType­: Type_ {
-    public var arguments: [(String?, Type_)]
+public struct FunctionType: Type_ {
+    public var arguments: [Type_]
```

Comment:
@koher Swift compiler doesn't accept labels for arguments in function type representation.
It only accepts `(Int, String) -> Void` or `(_ a: Int, _ b: String) -> Void`
Not `(a: Int, b: String) -> Void`.